src/runtime/storage/storage-manager.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {CRDTTypeRecord} from '../../crdt//lib-crdt.js';
+import {TypeToCRDTTypeRecord} from './storage.js';
+import {ActiveStore} from './active-store.js';
+import {Type} from '../../types/lib-types.js';
+import {StoreInfo} from './store-info.js';
+import {StorageService} from './storage-service.js';
+
+export interface StorageEndpointManager {
```

Comment:
At a comment explaining what a StorageEndpointManager is.

Reply:
done (and added TODOs for the next refactoring steps)